src/TodoApp.tsx

```diff
@@ -0,0 +1,124 @@
+/* eslint-disable jsx-a11y/control-has-associated-label */
```

Comment:
You should not disable eslint rules in your file. Consider removing the eslint-disable line and fixing the issue instead.